src/wallet/wallet.h

```diff
@@ -905,18 +905,25 @@ class CWallet final : public WalletStorage, public interfaces::Chain::Notificati
     /** Loads the flags into the wallet. (used by LoadWallet) */
     bool LoadWalletFlags(uint64_t flags);
 
-    /** Returns a bracketed wallet name for displaying in logs, will return [default wallet] if the wallet has no name */
-    std::string GetDisplayName() const override
+    /** Return wallet name for use in logs, will return "default wallet" if the wallet has no name. */
+    std::string LogName() const override
     {
-        std::string wallet_name = GetName().length() == 0 ? "default wallet" : GetName();
-        return strprintf("[%s]", wallet_name);
+        std::string name{GetName()};
+        return name.empty() ? "default wallet" : name;
```

Comment:
nit: perhaps a `static const` can be used
```suggestion
        return name.empty() ? DEFAULT_NAME : name;
```